web/src/components/Notifications.jsx

```diff
@@ -159,6 +162,63 @@ const autolink = (s) => {
   return <>{parts}</>;
 };
 
+const MarkdownContainer = styled("div")`
+  line-height: 1;
+
+  h1,
+  h2,
+  h3,
+  h4,
+  h5,
+  h6,
+  p,
+  pre,
+  ul,
+  ol {
+    margin: 0;
+  }
+
+  p {
+    line-height: 1.2;
+  }
+
+  blockquote {
+    margin: 0;
+    padding-inline: 1rem;
+    background: ${(theme) => (theme.mode === "light" ? "#f1f1f1" : "#aeaeae")};
+  }
+
+  ul,
+  ol {
+    padding-inline: 1rem;
+  }
+`;
+
+const MarkdownContent = ({ content }) => {
+  const [reactContent, setMarkdownSource] = useRemark();
+
+  useEffect(() => {
+    setMarkdownSource(content);
+  }, [content]);
+
+  return <MarkdownContainer>{reactContent}</MarkdownContainer>;
+};
+
+const NotificationBody = ({ notification }) => {
+  const markdownAlwaysEnabled = useLiveQuery(async () => prefs.markdownAlwaysEnabled());
+
+  // TODO: check notification content-type when implemented on the server
```

Comment:
You wanna implement this too? :grimacing: -- Branch out a little? Do a little Go :-D

If so, maybe rebase on top of my markdown branch?